packages/Rakefile

```diff
@@ -61,20 +61,8 @@ class GroongaPackageTask < PackagesGroongaOrgPackageTask
     ]
   end
 
-  def apt_prepare_debian_control_h3(control, target)
-    case target
-    when /\Aubuntu-focal/
-      enable_h3 = "#"
-    else
-      enable_h3 = ""
-    end
-    control.gsub(/@ENABLE_H3@/, enable_h3)
-  end
-
   def apt_prepare_debian_control(control_in, target)
```

Comment:
note: We should define `apt_prepare_debian_control` because it's inherited from `PackageTask`.
https://github.com/apache/arrow/blob/13940cdc7e463eff8001e05b86be62a7c0e4c189/dev/tasks/linux-packages/package-task.rb#L320-L323

Reply:
We don't need to define this by renaming `debian/control.in` to `debian/control`: https://github.com/apache/arrow/blob/13940cdc7e463eff8001e05b86be62a7c0e4c189/dev/tasks/linux-packages/package-task.rb#L309-L317

Reply:
fix: [24a0b26](https://github.com/groonga/groonga/pull/2218/commits/24a0b26b6d76255713dcb35f8094b9d16104bd3f) Thanks you, I missed it.